packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -237,10 +250,33 @@ export interface IContainerRuntimeOptions {
     disableIsolatedChannels?: boolean;
 }
 
+/**
+ * Options for container runtime.
+ */
+export interface IContainerRuntimeOptions {
```

Comment:
Isn't this a breaking change? You should probably add this to breaking.md.

Reply:
Added to BREAKING.md